diff --git a/vendor/ezsystems/demobundle/EzSystems/DemoBundle/Helper/MenuHelper.php b/vendor/ezsystems/demobundle/EzSystems/DemoBundle/Helper/MenuHelper.php
index a7cf6bd..7858bea 100644
--- a/vendor/ezsystems/demobundle/EzSystems/DemoBundle/Helper/MenuHelper.php
+++ b/vendor/ezsystems/demobundle/EzSystems/DemoBundle/Helper/MenuHelper.php
@@ -94,6 +94,7 @@ class MenuHelper
         $criteria = array(
             new Criterion\Subtree($rootLocation->pathString),
             new Criterion\Visibility(Criterion\Visibility::VISIBLE),
+            new Criterion\LanguageCode($this->repository->getContentLanguageService()->getDefaultLanguageCode())
         );
 
         if ($includeContentTypeIdentifiers) {
