diff --git a/ezpublish/EzPublishKernel.php b/ezpublish/EzPublishKernel.php
index 12f98d4..96ff101 100644
--- a/ezpublish/EzPublishKernel.php
+++ b/ezpublish/EzPublishKernel.php
@@ -52,6 +52,7 @@ class EzPublishKernel extends Kernel
             new EzSystems\EzPlatformSolrSearchEngineBundle\EzSystemsEzPlatformSolrSearchEngineBundle(),
             new EzSystems\EzPlatformXmlTextFieldTypeBundle\EzSystemsEzPlatformXmlTextFieldTypeBundle(),
             new AppBundle\AppBundle(),
+            new eZ\Bundle\EzPublishLegacyBundle\EzPublishLegacyBundle($this),
         );
 
         switch ($this->getEnvironment()) {
@@ -84,4 +85,19 @@ class EzPublishKernel extends Kernel
     {
         $loader->load($this->getRootDir() . '/config/config_' . $this->getEnvironment() . '.yml');
     }
+
+    protected function initializeContainer() {
+
+        parent::initializeContainer();
+
+        if (PHP_SAPI == 'cli') {
+
+            $this->getContainer()->enterScope('request');
+
+            $this->getContainer()->set('request', new \Symfony\Component\HttpFoundation\Request(), 'request');
+
+        }
+
+    }
+
 }
