diff --git a/ezpublish_legacy/kernel/private/classes/global_functions.php b/ezpublish_legacy/kernel/private/classes/global_functions.php
index 6d22e94..da1b408 100644
--- a/ezpublish_legacy/kernel/private/classes/global_functions.php
+++ b/ezpublish_legacy/kernel/private/classes/global_functions.php
@@ -84,28 +84,32 @@ function eZDisplayResult( $templateResult )
  *
  * @deprecated Since 5.0
  */
-function eZUpdateDebugSettings()
-{
-    $settings = array();
-    list( $settings['debug-enabled'], $settings['debug-by-ip'], $settings['log-only'], $settings['debug-by-user'], $settings['debug-ip-list'], $logList, $settings['debug-user-list'] ) =
-        eZINI::instance()->variableMulti(
-            'DebugSettings',
-            array( 'DebugOutput', 'DebugByIP', 'DebugLogOnly', 'DebugByUser', 'DebugIPList', 'AlwaysLog', 'DebugUserIDList' ),
-            array( 'enabled', 'enabled', 'disabled', 'enabled' )
-        );
-    $settings['always-log'] = array();
-    foreach (
-        array(
-            'notice' => eZDebug::LEVEL_NOTICE,
-            'warning' => eZDebug::LEVEL_WARNING,
-            'error' => eZDebug::LEVEL_ERROR,
-            'debug' => eZDebug::LEVEL_DEBUG,
-            'strict' => eZDebug::LEVEL_STRICT
-        ) as $name => $level )
+if (!function_exists('eZUpdateDebugSettings')) {
+
+    function eZUpdateDebugSettings()
     {
-        $settings['always-log'][$level] = $logList !== null && in_array( $name, $logList );
+        $settings = array();
+        list( $settings['debug-enabled'], $settings['debug-by-ip'], $settings['log-only'], $settings['debug-by-user'], $settings['debug-ip-list'], $logList, $settings['debug-user-list'] ) =
+            eZINI::instance()->variableMulti(
+                'DebugSettings',
+                array( 'DebugOutput', 'DebugByIP', 'DebugLogOnly', 'DebugByUser', 'DebugIPList', 'AlwaysLog', 'DebugUserIDList' ),
+                array( 'enabled', 'enabled', 'disabled', 'enabled' )
+            );
+        $settings['always-log'] = array();
+        foreach (
+            array(
+                'notice' => eZDebug::LEVEL_NOTICE,
+                'warning' => eZDebug::LEVEL_WARNING,
+                'error' => eZDebug::LEVEL_ERROR,
+                'debug' => eZDebug::LEVEL_DEBUG,
+                'strict' => eZDebug::LEVEL_STRICT
+            ) as $name => $level )
+        {
+            $settings['always-log'][$level] = $logList !== null && in_array( $name, $logList );
+        }
+        eZDebug::updateSettings( $settings );
     }
-    eZDebug::updateSettings( $settings );
+
 }
 
 /**
