diff --git a/vendor/ezsystems/legacy-bridge/mvc/View/TwigContentViewLayoutDecorator.php b/vendor/ezsystems/legacy-bridge/mvc/View/TwigContentViewLayoutDecorator.php
index b2a883f..6baba9f 100644
--- a/vendor/ezsystems/legacy-bridge/mvc/View/TwigContentViewLayoutDecorator.php
+++ b/vendor/ezsystems/legacy-bridge/mvc/View/TwigContentViewLayoutDecorator.php
@@ -10,12 +10,11 @@
 namespace eZ\Publish\Core\MVC\Legacy\View;
 
 use eZ\Publish\Core\MVC\ConfigResolverInterface;
-use eZ\Publish\Core\MVC\Symfony\View\ContentViewInterface;
 use eZ\Publish\Core\MVC\Symfony\View\ContentView;
 use eZ\Publish\Core\Base\Exceptions\InvalidArgumentType;
 use Twig_Environment;
 
-class TwigContentViewLayoutDecorator implements ContentViewInterface
+class TwigContentViewLayoutDecorator extends ContentView
 {
     /**
      * @var \eZ\Publish\Core\MVC\Symfony\View\ContentView
