diff --git a/vendor/ezsystems/legacy-bridge/bundle/Controller/PreviewController.php b/vendor/ezsystems/legacy-bridge/bundle/Controller/PreviewController.php
index df13109..bfaa351 100644
--- a/vendor/ezsystems/legacy-bridge/bundle/Controller/PreviewController.php
+++ b/vendor/ezsystems/legacy-bridge/bundle/Controller/PreviewController.php
@@ -14,6 +14,7 @@ use eZ\Publish\API\Repository\Values\Content\Location;
 use eZ\Publish\Core\MVC\ConfigResolverInterface;
 use eZ\Publish\Core\MVC\Symfony\Controller\Content\PreviewController as BasePreviewController;
 use eZ\Publish\Core\MVC\Symfony\SiteAccess;
+use Symfony\Component\HttpFoundation\Request;
 
 class PreviewController extends BasePreviewController
 {
@@ -30,9 +31,9 @@ class PreviewController extends BasePreviewController
         $this->configResolver = $configResolver;
     }
 
-    protected function getForwardRequest( Location $location, Content $content, SiteAccess $previewSiteAccess )
+    protected function getForwardRequest( Location $location, Content $content, SiteAccess $previewSiteAccess, Request $request, $language )
     {
-        $request = parent::getForwardRequest( $location, $content, $previewSiteAccess );
+        $request = parent::getForwardRequest( $location, $content, $previewSiteAccess, $request, $language);
         // If the preview siteaccess is configured in legacy_mode, we forward to the LegacyKernelController.
         if ( $this->configResolver->getParameter( 'legacy_mode', 'ezsettings', $previewSiteAccess->name ) )
         {
